meson.build

```diff
@@ -141,7 +142,7 @@ subdir('test')
 
 includes = include_directories('src')
 
-udevlibexecdir = join_paths(prefixdir, 'lib/udev')
+udevlibexecdir = join_paths(libdir, 'udev')
```

Comment:
If this is to be fixed, I think it should be fixed to use pkg-config, as udev ships this dir in its `.pc` file.